src/main/java/com/stories/controller/StoriesController.java

```diff
@@ -79,4 +81,13 @@ public void deleteStory(@Valid @PathVariable String id) throws Exception {
 	public StoryDomain updateStory(@Valid @RequestBody StoryDomain request, @PathVariable String id) throws Exception {
 		return storyService.updateStory(request, id);
 	}
+	
+		@ApiOperation(value = " GET Tasks ", notes = " This operation will return the tasks of a story ")
+		@ApiResponses({ @ApiResponse(code = 200, message = " Success operation ")})
+		@ResponseStatus(value = HttpStatus.OK)
+		@GetMapping(value = "/{id}/tasks", produces = "application/json")
+		@ResponseBody
+		public List<TasksDomain> findTasksByyId(@Valid @PathVariable String id) throws EntityNotFoundException{
```

Comment:
We could follow the standar of the names of storyAPI, could you please change the name like: getTasksByStory ?

Reply:
fixed